docs/installation/Binaries.md

```diff
@@ -0,0 +1,111 @@
+# Binary package installation
+
+Released versions are available at [GitHub](https://github.com/TouK/nussknacker/releases).
+
+Please note, that while you can install Designer from `.tgz` with Lite engine configured, you still
+need configured Kubernetes cluster to actually run scenarios in this mode - we recommend using Helm installation for that mode.
+
+## Prerequisites
+
+We assume that `java` (recommended version is JDK 11) is on path.
+
+Please note that default environment variable configuration assumes that Flink, InfluxDB, Kafka and Schema registry are
+running on `localhost` with their default ports configured. See [environment variables](../configuration/Common.md#environment-variables) section
+for the details. Also, `GRAFANA_URL` is set to `/grafana`, which assumes that reverse proxy
+like [NGINX](https://github.com/TouK/nussknacker-quickstart/tree/main/docker/common/nginx) is used to access both Designer and
+Grafana. For other setups you should change this value to absolute Grafana URL.
+
+`WORKING_DIR` environment variable is used as base place where Nussknacker stores its data such as:
+
+- logs
+- embedded database files
+- scenario attachments
+
+## Startup script
+
+We provide following scripts:
+
+- `run.sh` - to run in foreground, it's also suitable to use it for systemd service
+- `run-daemonized.sh` - to run in background, we are using `nussknacker-designer.pid` to store PID of running process
+
+## File structure
```

Comment:
is this not necessary outside the context of installation through binary? In Docker we also refer to `NUSSKNACKER_DIR` but its not explained in that context.

Reply:
It's not so important in the context of Docker because you don't have to configure it on your own. I'd leave it as it is now.